server.js

```diff
@@ -375,7 +375,7 @@ app.get('/v0/list/*', AuthRequired, function(req, res) {
 
     var mailOptions = {
         from: 'help@modelseed.org',
-        to: 'help@modelseed.org',         // list of receivers
+        to: 'qzhang@anl.gov',         // list of receivers
```

Comment:
Are you actively using this for your testing? I want to be careful that we don't forget to revert the email address when deploying, so can you revert this change? Thanks.

Reply:
Yes.  That's my noted point.  For now, I don't want to bombard people on the list of help@modelseed.org before we decide to deploy it into prod.

Reply:
I understand that, but my point is that you don't need to include this in the PR, you can change it locally.